mmv1/third_party/terraform/services/netapp/resource_netapp_storage_pool_test.go.erb

```diff
@@ -41,6 +41,15 @@ func TestAccNetappStoragePool_storagePoolCreateExample_update(t *testing.T) {
 				ImportStateVerify:       true,
 				ImportStateVerifyIgnore: []string{"location", "name", "labels", "terraform_labels"},
 			},
+			{
+				Config: testAccNetappStoragePool_storagePoolCreateExample_autoTieredPool(context),
```

Comment:
Since this is immutable, it should get covered in single-step test, probably a new one!

Reply:
Done